src/alert-manager/deploy/service.yaml

```diff
@@ -37,4 +38,4 @@ upgraded-script: upgraded.sh
 
 
 deploy-rules:
-  - in: pai-master
\ No newline at end of file
+  - in: pai-master
```

Comment:
The `deploy-rule` doesn't apply for CronJob, https://github.com/microsoft/pai/blob/master/deployment/paiLibrary/paiService/service_template_generate.py#L61
 we can add a nodeSelector in the CronJob definition.